### PR TITLE
chore: prep for 3.1.0 release

### PR DESCRIPTION
**Pull request by kgriffs**
# Summary of Changes

Add RELEASE.md and prep for 3.1.0rc1

### CODE REVIEW

**Review by vytas7 — COMMENTED** [review]
Looks great 👍
You haven't removed the news fragments yet. Do we want to keep them for some time even though the report has been rendered?

**Comment by kgriffs** [comment]
Oops, forgot to rm the news fragment fikes.

File: RELEASE.md [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+key changes in this release.
+
+Next update the contributors and render towncrier fragments by running:
+
+    $ tox -e changelog_release
+
+Examine the updated RST as well as the rendered HTML docs and make any
+adjustments as needed. On OS X it's as simple as:
+
+.. code:: bash
```
Comment (vytas7) [inline RELEASE.md:69]: A cosmetic nitpick: these `.. code::` directives seem to have been copypasted from `.rst`, and they don't port to Markdown verbatim. Easy to check with "View file" right herę in GH.

**Review by vytas7 — CHANGES_REQUESTED** [review]
LGTM sans removal of newsfragments :100:
(And there are a couple of non-critical sphinx-isms in `RELEASE.md`.)

File: RELEASE.md (hunk 1 above)
Comment (kgriffs) [reply RELEASE.md:69]: oh snap, forgot for some reason I wasn't in RST. will fix

Comment (kgriffs) [reply RELEASE.md]: done

**Review by vytas7 — DISMISSED** [review]
👍

**Review by vytas7 — APPROVED** [review]

File: RELEASE.md [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+
+```sh
+$ open docs/_build/html/changes/index.html
+```
+
+Or on Linux:
+
+```sh
+$ xdg-open docs/_build/html/changes/index.html
+```
```
Comment (CaselIT) [inline RELEASE.md:79]: we probably don't need to add it here, but on windows you literally just type the filename and it opens in the default program

**Review by CaselIT — APPROVED** [review]

File: RELEASE.md (hunk 2 above)
Comment (vytas7) [reply RELEASE.md:79]: Probably it's irrelevant for `RELEASE.md`, but we could add this to other documents. But isn't this dependent on which Windows shell you use, like `cmd.exe` vs PowerShell? Or is it the same behaviour on all of them?

Comment (CaselIT) [reply RELEASE.md:79]: they both do the same, ie open the default program. this is not limited to html file, py or md files work the same

**Pull request merged**